hack/cluster/cluster.go

```diff
@@ -44,6 +44,10 @@ func run(ctx context.Context, log *logrus.Entry) error {
 	}
 	clusterName := os.Getenv(Cluster)
 
+	if clusterName == "" {
+		return fmt.Errorf("your environment variable \"CLUSTER\" is empty, please pass a value")
+	}
+
```

Comment:
I think we can use the already defined variable name Cluster instead of hardcoding it on the error message, I would also change a bit the error message, to something like
```go
if clusterName == "" {
    return fmt.Errorf("the environment variable %v is empty, please use a non empty value", Cluster)
}
```

I also think it would be a good idea to add a similar check in pkg/util/cluster/cluster.go:144 as this funcitonality can be potentially called from other places than the hack script as it is a separate component of a different package.

Something like (pkg/util/cluster/cluster.go:143):
```go
func (c *Cluster) Create(ctx context.Context, vnetResourceGroup, clusterName string, osClusterVersion string) error {
if clusterName == "" {
    return fmt.Errorf("clusterName can not be empty")
}
. . .
```

Reply:
how do you feel like move all validations to one place to  **ValidateVars()**, like:

```
		if v, found := os.LookupEnv(v); !found || v == "" {
			return fmt.Errorf("environment variable %q unset or set to empty", v)
		}
```

Reply:
This is a good idea If we are sure that we are not changing the behavior of the code already using ValidateVars().
😃